osu.Game/Skinning/LegacySkin.cs

```diff
@@ -52,22 +53,29 @@ protected override void Dispose(bool isDisposing)
 
         public override Drawable GetDrawableComponent(string componentName)
         {
+            bool animatable = false;
```

Comment:
seems to be true on most components, would be easier to set it true on default and change its value on non-animatable components (`Play/osu/number-text` in this case)

Reply:
This method is called for more components than the switch specifies (e.g. cursor or background) which shouldn't be animated